addons/website_forum/controllers/main.py

```diff
@@ -135,6 +135,7 @@ def questions(self, forum=None, tag=None, page=1, filters='all', my=None, sortin
             'my': my,
             'author_id': author.id if author else False,
             'include_answers': include_answers,
+            'my_profile': request.env.user == author,
```

Comment:
And this my previous comment is especially true here. author will be falsy in your current code if int(author) == request.env.user.id. So 'my_profile will never be True